src/orquestra/sdk/_base/_driver/_ce_runtime.py

```diff
@@ -249,7 +255,32 @@ def stop_workflow_run(self, workflow_run_id: WorkflowRunId) -> None:
         try:
             self._client.terminate_workflow_run(workflow_run_id)
         except _exceptions.InvalidTokenError as e:
-            raise exceptions.UnauthorizedError(f"{e}")
+            raise exceptions.UnauthorizedError(f"{e}") from e
+
+    def list_workflow_runs(
+        self,
+        *,
+        limit: Optional[int] = None,
+        prefix: Optional[str] = None,
+        max_age: Optional[timedelta] = None,
+        state: Optional[Union[State, List[State]]] = None,
+    ) -> List[WorkflowRun]:
+        """
+        List the workflow runs, with some filters
+
+        Args:
+            limit: Restrict the number of runs to return, prioritising the most recent.
+            prefix: Only return runs that start with the specified string.
+            max_age: Only return runs younger than the specified maximum age.
+            status: Only return runs of runs with the specified status.
+        Returns:
+                A list of the workflow runs
+        """
+        try:
+            runs = self._client.list_workflow_runs()
```

Comment:
**suggestion**: as of today the filters aren't passed to the `client`. I suppose we will pass them in the future. Is there a ticket we can mention in a comment here?

Reply:
done in [7445df6](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/3/commits/7445df611c436a48b68610f086eb06f3e23b5c5d)